include/InputParser.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+class InputParser
+{
+public:
+	static std::vector<std::string> tokenize(const std::string& input);
```

Comment:
Since this class has just one static function, maybe it shouldn't be a class and be just a single function instead